wagtail/admin/tests/test_compare.py

```diff
@@ -1145,7 +1145,7 @@ def test_has_changed(self):
             objs_a, objs_b
         )
         self.assertEqual(map_forwards, {0: 1})  # Map Father Christmas to Father Ted
-        self.assertEqual(map_backwards, {1: 0})  # Map Father Ted ot Father Christmas
+        self.assertEqual(map_backwards, {1: 0})  # Map Father Ted to Father Christmas
```

Comment:
Just wanted to make sure if this was done on purpose since it's `map_backwards` but it wouldn't make sense to re-order the letters in the word itself, just it's chronological order in the array

Reply:
As long as tests still pass - makes sense.